systemd/ceph-osd@.service

```diff
@@ -1,6 +1,6 @@
 [Unit]
-Description=Ceph object storage daemon
-After=network-online.target local-fs.target time-sync.target
+Description=Ceph object storage daemon osd.%i
```

Comment:
This line is not changed in the master commit, so it should not be changed in the cherry-pick either.

Also, your commit message contains no description of the conflict or how you resolved it.